requirements.txt

```
@@ -3,11 +3,12 @@ importlib_metadata>=4.8.1
 qiskit-terra>=0.19.2
 qiskit-aer>=0.10.3
 qiskit-ibmq-provider>=0.18.3
-qiskit-optimization>=0.3.0
+qiskit-optimization[cplex]>=0.3.0
```

```suggestion
qiskit-optimization>=0.3.0
```

Reverting this change since it is actually [redundant](https://github.com/Qiskit/qiskit-optimization/blob/ea045123fdaeaed3789731ef01160fca0f8db4a1/setup.py#L72) with having specified `cplex` explicitly later in the file.  That way is preferred, because it at least allows us to specify a minimum version.